tools/enso4igv/src/main/java/org/enso/tools/enso4igv/EnsoSbtClassPathProvider.java

```diff
@@ -349,7 +352,12 @@ record EnsoSources(
     ) implements SourceGroup {
         @Override
         public FileObject getRootFolder() {
-            return srcCp.getRoots()[0];
+            var arr = srcCp.getRoots();
+            if (arr.length == 0) {
+                LOG.log(Level.SEVERE, "srcCp is empty for {0}", this);
```

Comment:
```suggestion
                LOG.log(Level.SEVERE, "Source classpath is empty for {0}", this);
```

Would it make for a more readable error message?